internal/k8s/validation_test.go

```diff
@@ -3420,8 +3420,10 @@ func TestValidatePath(t *testing.T) {
 		"/abc}{abc",
 	}
 
+	pathType := networking.PathTypeExact
+
 	for _, path := range validPaths {
-		allErrs := validatePath(path, field.NewPath("path"))
+		allErrs := validatePath(path, &pathType, field.NewPath("path"))
```

Comment:
we don't have any tests with `Prefix`?

Reply:
`validatePath` only has different behaviour if the `pathType` is set `ImplementationSpecific`, running the tests with `Prefix` is identical to running with `Exact` (we don't check which it is for this validation test), so there wouldn't be anything new checked.